zeppelin-web/src/app/interpreter/interpreter.html

```diff
@@ -173,8 +173,8 @@ <h3 class="interpreter-title">{{setting.name}}
       <br />
       <div class="col-md-12">
         <div class="checkbox">
-          <span class="input-group">
-            <label><input type="checkbox" style="width:0%;height:0%" id="isExistingProcess" ng-model="setting.option.isExistingProcess" ng-disabled="!valueform.$visible"/>
+          <span class="input-group" style="line-height:30px;">
+            <label><input type="checkbox" style="width:10%" id="isExistingProcess" ng-model="setting.option.isExistingProcess" ng-disabled="!valueform.$visible"/>
```

Comment:
Because without width, it will be as following image.
![image](https://cloud.githubusercontent.com/assets/3348133/17388173/5c93b218-5a35-11e6-8bab-c9360d033063.png)